app/src/main/res/layout/fragment_passcode.xml

```diff
@@ -215,6 +216,22 @@
                 app:layout_constraintEnd_toEndOf="parent"
                 app:layout_constraintStart_toStartOf="parent" />
 
+
+            <TextView
+                android:id="@+id/show_keyboard"
+                style="@style/TextLink"
+                android:layout_width="256dp"
```

Comment:
why does it have fixed margins and dimensions? Layout should be responsive.
Also please make sure this label won't be pushed up when keyboard appears -> keyboard should be displayed on top of this label.